### ISSUE TITLE
MariaDB 10.3 repo not available anymore

### ISSUE THREAD

**Issue by m-witt** [plesk/centos2alma#152]
Since MariaDB 10.3 is not available anymore from official MariaDB yum repositorys. Will it affect the update process or will for example 10.4 automatically be chosen ?

Repo URL: 
http://mirror.mariadb.org/yum/10.3/

**Comment by SandakovMM** [comment]
It will probably be a problem. Let me check, please.

**Comment by m-witt** [comment]
I've tested myself and it seems to be a problem : 
```
2023-12-20 14:35:34,229 - INFO - Start stage 5.
2023-12-20 14:35:34,262 - INFO - Skipped: updating mariadb databases!
2023-12-20 14:35:34,294 - INFO - Do: update modern mariadb!
2023-12-20 14:35:34,356 - ERROR - Failed: update modern mariadb!. The reason: 'NoneType' object is not subscriptable
2023-12-20 14:35:35,799 - ERROR - centos2alma process has been failed. Error: Failed: update modern mariadb!. The reason: 'NoneType' object is not subscriptable
```

CentOS 7 + MariaDB 10.3

Currently blocking all usage of this Software

**Comment by SandakovMM** [comment]
Could you please share a feedback archive? It could be created by calling `centos2alma --prepare-feedback'

**Comment by m-witt** [comment]
> Could you please share a feedback archive? It could be created by calling `centos2alma --prepare-feedback'

[centos2alma_feedback.zip](https://github.com/plesk/centos2alma/files/13751497/centos2alma_feedback.zip)

**Comment by SandakovMM** [comment]
Thank you. I am currently looking into solutions for the problem. I hope to have it resolved within the next two weeks

**Comment by ManfredLan** [comment]
Just chiming in here, we're having the same issue, but on CentOS7 with MariaDB 10.6 already installed.

**Comment by SandakovMM** [comment]
I have resolved the issue in #153. It was caused by unexpected items in the leapp json configuration. You can download the new version of centos2alma from the following link: [centos2alma.zip](https://github.com/plesk/centos2alma/files/13786375/centos2alma.zip).
I plan to release the new version tomorrow.

**Comment by SandakovMM** [comment]
Release has been [published](https://github.com/plesk/centos2alma/releases/tag/v1.2.2)
